### PR TITLE
UHF-8478: Remove deprecated helfi_rekry_job_search module

### PR DESCRIPTION
**Pull request by xkhaven**
# [UHF-8478](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8478)

## What was done

* Removed deprecated `field_copied` field from node type `job_listing`
* Removed deprecated module `helfi_rekry_job_search`

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-8478_Remove_field_copied`
  * `make fresh`
* Run `make drush-cr`

## How to test

* [x] Node type `job_listing` doesn't have field `field_copied`
* [x] Module `helfi_rekry_job_search` is removed
* [ ] Check that code follows our standards

## Designers review

* [X] This PR does not need designers review

## Other PRs

* https://github.com/City-of-Helsinki/drupal-hdbt/pull/824

[UHF-8478]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8478?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Jussiles — CHANGES_REQUESTED** [review]
Looks good, but I added couple of comments that should be changed. Also functions from module file should be moved under `helfi_rekry_content.module`

File: conf/cmi/search_api.index.job_listings.yml [hunk 1]
```diff
@@ -323,6 +314,7 @@ datasource_settings:
 processor_settings:
   add_url: {  }
   aggregated_field: {  }
+  custom_value: {  }
```
[the remark this answers is not in the document]
Comment (xkhaven) [reply conf/cmi/search_api.index.job_listings.yml]: Removed this line

File: conf/cmi/helfi_rekry_content.job_listings.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 search_page: '2971'
-redirect_403_page: '2971'
+redirect_403_page: '4848'
```
[the remark this answers is not in the document]
Comment (xkhaven) [reply conf/cmi/helfi_rekry_content.job_listings.yml]: Reverted this config

File: public/modules/custom/helfi_rekry_job_search/helfi_rekry_job_search.module [hunk 3]
```diff
@@ -1,109 +0,0 @@
...
-
-use Drupal\Core\Language\LanguageInterface;
-use Drupal\helfi_platform_config\DTO\ParagraphTypeCollection;
-
-/**
- * Implements hook_page_attachments_alter().
- *
- * Update job search's task area pages metadata.
- */
-function helfi_rekry_job_search_page_attachments_alter(array &$attachments) {
```
[the remark this answers is not in the document]
Comment (xkhaven) [reply public/modules/custom/helfi_rekry_job_search/helfi_rekry_job_search.module:18]: These codes are now copied to `helfi_rekry_content.module`

**Review by Jussiles — APPROVED** [review]
Looks good now! 🦜

**Pull request merged**